users/class_tests/login_tests.py

```diff
@@ -4,6 +4,8 @@
 from django.contrib.auth import authenticate
 from pytest_django.asserts import assertTemplateUsed
 from users.forms import MyUserCreationForm as LoginForm
+from .constants import VALID_USERNAME, VALID_EMAIL, VALID_PASSWORD
+from .constants import NOT_VALID_EMAIL, NOT_VALID_PASSWORD, LOGIN_HTML_PATH
```

Comment:
instead of repeating the "from .constants" 2 times, you can do it like:
```suggestion
from .constants import (
    VALID_USERNAME, VALID_EMAIL, VALID_PASSWORD, NOT_VALID_EMAIL, 
    NOT_VALID_PASSWORD, LOGIN_HTML_PATH
)
```

Reply:
done.

Reply:
Not fixed yet